usaspending_api/awards/models.py

```diff
@@ -384,12 +405,27 @@ def get_default_fields():
         default_fields = AwardAction.get_default_fields()
         return default_fields + [
             "type",
+            "type_description",
             "cost_or_pricing_data",
+            "type_of_contract_pricing",
+            "type_of_contract_pricing_description",
             "naics",
             "naics_description",
             "product_or_service_code"
         ]
 
+    def get_pricing_type_description(self):
+        description = [item for item in CONTRACT_PRICING_TYPES if item[0] == self.type_of_contract_pricing]
+        if len(description) == 0:
+            return "Unknown Type"
```

Comment:
Sure, I can add that. I didn't because 'U' is actually a used type code for contract pricing, so I'll have to use a different string. (It is expected to get this data every time)